backend/pkg/controllers/main.controller.go

```diff
@@ -279,7 +327,7 @@ func (c *MainController) RateIssue(ctx *gin.Context) {
 	}
 
 	//TODO: Remove hardcoded userId
-	userId = "4c78e05c-2f83-4e6e-b4c1-8721618a1c89"
+	// userId = "4c78e05c-2f83-4e6e-b4c1-8721618a1c89"
```

Comment:
We can remove it, if we don't use it anymore

Reply:
I will leave it as testing account id at least till I help you make changes in your setups to use correct ids even with postman